app/routes/public/cfs.js

```diff
@@ -1,15 +1,21 @@
 import Ember from 'ember';
+import moment from 'moment';
 
-const { Route, RSVP } = Ember;
+const { Route, RSVP, computed } = Ember;
 
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Call for Speakers');
   },
 
   model() {
+    const eventDetails = this.modelFor('public');
     return RSVP.hash({
-      event: this.modelFor('public')
+      event        : eventDetails,
+      speakersCall : eventDetails.get('speakersCall'),
+      isOpen       : computed('eventDetails.endsAt', 'speakersCall.endsAt', function() {
+        return moment(this.get('eventDetails.endsAt')).isAfter(this.get('speakersCall.endsAt'), 'year');
```

Comment:
@harshitagupta30 it's alright you don't have to apologise. Its okay to make mistakes. Maybe if you could tell me how/why you used this logic, I could try to explain why it isn't right. (In case you had mis-understood the call for speakers logic) :)